<?xml version="1.0"?>
<launch>
  	
	  <!-- debug settings -->
	<arg name="plan_movement_debug"			default="true"/>

	<node name="plan_movement"
			pkg="flex_grasp"
			type="plan_movement_node.py"
			output="screen">
			<param name ="debug" value="$(arg plan_movement_debug)"/>
	</node>
</launch>

